src/ui/components/stepCard/StepCard.native.test.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react'
+
+import { render, screen } from 'tests/utils'
+import { BicolorAroundMe } from 'ui/svg/icons/BicolorAroundMe'
+
+import { StepCard } from './StepCard'
+
+describe('<StepCard />', () => {
+  it('should render title and icon', () => {
+    const title = 'This is a StepCard'
+    const icon = <MockIcon />
+    render(<StepCard title={title} icon={icon} />)
```

Comment:
Je ne comprends pas ton `<MockIcon />` ?

Pourquoi ne pas simplement faire 
```suggestion
    render(<StepCard title={title} icon={<BicolorAroundMe />} />)
```